Makefile

```diff
@@ -6,8 +6,9 @@ NETLIFY_FUNC      = $(NODE_BIN)/netlify-lambda
 # but this can be overridden when calling make, e.g.
 # CONTAINER_ENGINE=podman make container-image
 CONTAINER_ENGINE ?= docker
+IMAGE_REGISTRY ?= gcr.io/k8s-staging-sig-docs
 IMAGE_VERSION=$(shell scripts/hash-files.sh Dockerfile Makefile | cut -c 1-12)
```

Comment:
Maybe we should update this to include `_GIT_TAG` information?

Reply:
Do you mean something like : 
```
TAG = v$(HUGO_VERSION)-$(IMAGE_VERSION
CONTAINER_IMAGE   = $(IMAGE_REGISTRY)/k8s-website-hugo:$(TAG)
```

Reply:
I'd consider including the commit ID in `$(TAG)` if building in the cloud, and defaulting to that for local image selection too.

Reply:
so the reason this does not have _GIT_TAG is if you don't have access to the registry it gracefully degrades to building an identically tagged image from the same sources (by hashing them) automatically. (think: users behind GFW).

(I think I discussed that more in the PR introducing it).